src/remove_locks.py

```diff
@@ -32,7 +32,10 @@
 if cursor.rowcount==0:
     print "Script is locked. Removing lock."
 
-    cursor.execute("insert into system (key,value,pid) values ('run_end',%s,%s)", (datetime.datetime.now(),str(pid),)
+    date_time_string = "%s" % (datetime.datetime.now())
+    date_time_string_fixed = date_time_string[:10] + 'T' + date_time_string[11:]
```

Comment:
`datetime.isoformat()` would be a better option for this - https://docs.python.org/2/library/datetime.html#datetime.datetime.isoformat

Reply:
Done.